core/storage/base_model/gae_models.py

```diff
@@ -54,10 +54,25 @@
 )
 
 EXPORT_POLICY = utils.create_enum(  # pylint: disable=invalid-name
+    # Indicates that a model's field is to be exported.
     'EXPORTED',
+    # Indicates that the field is exported as the Takeout id.
+    'EXPORTED_AS_MODEL_TAKEOUT_ID',
```

Comment:
Could you please add more detail about what exactly a "takeout ID" is? I don't understand the term.

Reply:
discussed offline, will create a separate test for validating that there is a single id designated as the EXPORTED_AS_MODEL_TAKEOUT_ID, and then get rid of the list that we use earlier (instead making the assumption that we have a single field that is the "takeout id")

Reply:
switch to EXPORTED_AS_KEY_FOR_TAKEOUT_DICT

Reply:
Or: EXPORTED_AS_KEY_FOR_TAKEOUT_DICT (and get rid of usage of "takeout ID" everywhere)

Reply:
Done